gridstatus/ieso.py

```diff
@@ -0,0 +1,307 @@
+import io
+import time
+import xml.etree.ElementTree as ET
+
+import pandas as pd
+import requests
+
+from gridstatus import utils
+from gridstatus.base import ISOBase, NotSupported
+from gridstatus.decorators import support_date_range
+from gridstatus.gs_logging import log
+
+# Load goes back several decades, with each year in one file.
+EARLIEST_LOAD = pd.Timestamp("2002-05-01").date()
+LOAD_INDEX_URL = "http://reports.ieso.ca/public/DemandZonal/"
```

Comment:
The hourly demand available here is only published once a day, so you can't get values for today until tomorrow. If we want real time data (in 5-minute intervals), we'd need to scrape the data here: http://reports.ieso.ca/public/RealtimeMktTotals/

Reply:
the thing that confuses me is I dont see how that file matches up with the real time graph on the status page

Reply:
Yeah, you're right. The data from the Realtime Markets Totals doesn't match what's on the graph.